### ISSUE TITLE
NFL Teams object returns 404 error

### ISSUE THREAD

**Issue by brandonschein** [roclark/sportsreference#254]
**Describe the bug**
  - Attempting to use NFL package's Teams object or any of its methods returns an HTTP Error 404

**To Reproduce**
```python
from sportsreference.nfl.teams import Teams

for team in Teams():
     print(team.name)

```

**Expected behavior**
  - Print all NFL team names

**Screenshots**
<img width="1119" alt="Screen Shot 2019-11-14 at 1 09 05 PM" src="https://user-images.githubusercontent.com/21065605/68883894-f5d3e600-06df-11ea-8ad1-641686a89c5c.png">

**Desktop (please complete the following information):**
 - OS: macOS Catalina 10.15.1
 - Sportsreference Version 0.46
 - Python Version 3.7

**Additional context**
 - API code references https://www.pro-football-reference.com/years/2019.html
 - https://www.pro-football-reference.com/years/2019 seems to work correctly

**Comment by oneshotbob** [comment]
I ran into the same issue

**Comment by DavidLiuGit** [comment]
I had the same issue. It's been fixed in #250.

**Comment by shafty80** [comment]
i am updated to the current version, and the issue is not fixed.

thanks

**Comment by roclark** [comment]
Hey @shafty80, can you provide some more details on this? Are you using the latest wheel available on PyPI (version 0.4.6), or are you using the master branch of the code? If the former, the update is not included in this release yet as I am planning on a few more patch fixes to be completed before pushing the next version.

If, however, you are using the latest code, this is something that I will need to look into.

**Comment by roclark** [comment]
I'm going to close this one for now as it should already be resolved with #250 as @DavidLiuGit mentioned, but if that appears to not be the case, please let me know and I will gladly reopen this for further troubleshooting!
